zerver/tests/test_home.py

```diff
@@ -155,6 +155,8 @@ def test_home(self) -> None:
             "subscriptions",
             "test_suite",
             "timezone",
+            "total_uploads_size",
```

Comment:
This is a rebase bug, I think; the other feature here was removed.

Reply:
@timabbott OK, I removed this line. Thanks!